src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,366 @@
+import {
+  VaAlert,
+  VaCheckboxGroup,
+  VaModal,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { scrollAndFocus } from 'platform/utilities/ui';
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  conditionsDescription,
+  conditionsPageTitle,
+  conditionsQuestion,
+  makeTEConditionsUISchema,
+} from '../../content/toxicExposure';
+import {
+  deletedToxicExposureAlertConfirmationContent,
+  deleteToxicExposureModalContent,
+  deleteToxicExposureModalDescription,
+  deleteToxicExposureModalTitle,
+} from '../../content/toxicExposureChoiceContent';
+import { formTitle, showToxicExposureDestructionModal } from '../../utils';
+
+export const toxicExposureChoicePageTitle =
+  'Option to claim conditions related to toxic exposure';
+
+/**
+ * List of all toxic exposure related keys that need to be deleted
+ * when user opts out of toxic exposure claims.
+ * @constant {string[]}
+ */
+export const toxicExposureKeys = [
```

Comment:
Seems like this could be a shared `const` with that included in src/applications/disability-benefits/all-claims/utils/submit.js.